queries/date.py

```diff
@@ -190,10 +192,35 @@ def help_text(lemma: str) -> str:
     | "morgundagurinn"
 
 QDateTomorrow →
-    "á_morgun"  # "á" "morgun"
+    "á_morgun" | "í" "fyrramálið"
+
+QDatePrevDay →
+    "dagsetning" QDateYesterday
+    | "dagsetningin" QDateYesterday
+    | "hvað" "var" "dagsetningin" QDateYesterday
+    | "hver" "var" "dagsetningin" QDateYesterday
```

Comment:
I can see that it would be useful to allow parenthesis in the grammar:

`("hvað" | "hver") "var" "dagsetningin"`

Reply:
Yes, that would be very useful for conciseness and expressiveness. Often wished it was available.